### DISCUSSION TITLE
i am unable to login to putty. Showing access denied

### DISCUSSION THREAD

**Discussion by shouryagoud** [learndevopsonline/learndevopsonline#1572] [Q&A] [answered]
![Screenshot 2023-11-05 121823](https://github.com/learndevopsonline/learndevopsonline/assets/114072857/8bef757e-e26f-4cb7-bff6-b4f3c65490b5)

**Comment by sreedharm07** [comment, marked as answer]
Its logged in for me. Password is DevOps321 d and o capital letters. Give password properly

**Reply by shouryagoud** [reply]
Sure. I tried 4 to 5 times. might be missed the Caps All the time. I will try it.Thanq

**Comment by shouryagoud** [comment]
![Screenshot 2023-11-05 163954](https://github.com/learndevopsonline/learndevopsonline/assets/114072857/124f61ec-ec68-43e9-ba25-0cc9ddb0f829)

**Reply by shouryagoud** [reply]
can you help in this

**Reply by sreedharm07** [reply]
can you please turn on your machine and share ip

**Reply by shouryagoud** [reply]
Frontend public ip 54.159.66.158 private ip 172.31.34.165
mysql public ip  18.215.232.206 private ip 172.31.35.52
backend public ip  54.196.67.154  private ip 172.31.36.43

**Reply by sreedharm07** [reply]
closing flower bracket missing 
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/fa4bf9bf-600a-4446-b2ae-21d27f4d797a)

**Reply by shouryagoud** [reply]
Oh Ok. Let me try once.

**Reply by shouryagoud** [reply]
Hi, actually i am trying to buy the domain and when it shown in the demo video it shows the cost 75Rs for one year and now for me it shows 89Rs and i proceed with that payment with upi and notification got for 10000Rs. So how can i deal with this

**Reply by sreedharm07** [reply]
please rise new discussion for new issue, already in this discussion 2 issues raised and now its 3 rd one.
dont use only one discussion for one 1+ issues. because the main issue of this discussion was resolved and we think this discussion was solved and we dont open this discussion. so please do it from next onwards

price varies according to domain name  you choose.
ex - for .com it is different cost, for .in it is different cost and you choose which is low cost

**Reply by shouryagoud** [reply]
Sorry i don't aware of that. Sure i will use new discussion and i use .online for the domain name

**Reply by sreedharm07** [reply]
yes use .online  because it costs less

**Reply by shouryagoud** [reply]
ya used that one after the process  when  redirected to payment process it shows the amount 10k to complete

**Reply by sreedharm07** [reply]
pls send screenshot
